airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -650,27 +642,17 @@ def _start_task(self, context):
         self.arn = response["tasks"][0]["taskArn"]
         self.log.info("ECS task ID is: %s", self._get_ecs_task_id(self.arn))
 
-        if self.reattach:
-            # Save the task ARN in XCom to be able to reattach it if needed
-            self.xcom_push(context, key=self.REATTACH_XCOM_KEY, value=self.arn)
```

Comment:
this is a somewhat-breaking change, as the example code
https://github.com/apache/airflow/blob/e0f21f43c63b13fd48f55aa660746edc37df1458/tests/system/providers/amazon/aws/example_ecs_fargate.py#L125-L126
was recommending setting reattach to true to get the ARN.

I think this sucked, but removing the arn entirely from the xcom values is not good either.
What we could do is set it all the time now that we d'ont rely on this anymore to know if we need to reattach.

Reply:
I guess this part was never work as it expected

Reply:
well, pushing the ARN to xcom at least was working.
I opened a PR to restore that specific thing.